src/argilla/client/feedback/integrations/sentencetransformers.py

```diff
@@ -0,0 +1,234 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import warnings
+from typing import List, Optional, Union
+
+from argilla.client.feedback.dataset.local.dataset import FeedbackDataset
+from argilla.client.feedback.dataset.remote.dataset import RemoteFeedbackDataset
+from argilla.client.feedback.schemas.records import FeedbackRecord
+from argilla.client.feedback.schemas.remote.records import RemoteFeedbackRecord
+from argilla.client.feedback.schemas.vector_settings import VectorSettings
+from argilla.utils.dependency import require_dependencies
+
+
+class SentenceTransformersExtractor:
+    """This class extracts a number of basic text descriptives from FeedbackDataset
+    records using the TextDescriptives library and adds them as record metadata."""
+
+    def __init__(
+        self,
+        model: Optional[Union["SentenceTransformer", str]] = "TaylorAI/bge-micro-v2",
```

Comment:
```suggestion
        model: Union["SentenceTransformer", str] = "TaylorAI/bge-micro-v2",
```

This cannot be `None`, so no `Optional`